public/javascript/allergen-filter.js

```diff
@@ -29,7 +29,7 @@ async function vegetarian() {
 }
```

Comment:
This is the fix I'm adding to mitigate the issue where filters don't work on userprofile view